deploy-board/deploy_board/templates/hosts/host_details.html

```diff
@@ -42,11 +42,21 @@
 
 {% if agent_wrappers %}
 <div class="panel panel-default">
+    {% if has_host_env_sidecar_agents %}
+    {% include "panel_heading.tmpl" with panel_title="Sidecar Agent Details" panel_body_id="sidecarAgentDetailsId" direction="down" copy_host_name_button=True %}
+    <div id="sidecarAgentDetailsId" class="collapse in panel-body">
+        <span>This host has relevant sidecars in it's environment. These have been marked with <span class="bg-success">green backgrounds</span> for visibility.</span>
```

Comment:
Nit: This is a sidecar deployment on the host. 
I think instead of highlight green (which can infer that the deployment was succeeded), we can do in blue and make the text, eg.`Environment: beacon/prod-aws-us-east-1a Deploy: 86fEpuN8S3yMY9kri9eStw` bold.

Reply:
Thanks, makes sense. Updated the text and the styling and attached the new screenshots above.